test/GlobStream.js

```diff
@@ -6,6 +6,16 @@ var concat = miss.concat;
 var pipe = miss.pipe;
 
 describe('GlobStream', function() {
+  it('emits an error if it is not first in the stream', function(done) {
+    var gs1 = stream('./fixtures/whatsgoingon/**/*.txt', [], { cwd: __dirname, });
+    var gs2 = stream('./fixtures/whatsgoingon/**/*.txt', [], { cwd: __dirname, });
+
+    pipe([
+      gs1,
+      gs2,
+    ]);
+  });
```

Comment:
I can't really figure out a proper way to handle it either (I thought it would emit an error instead of just throw async).

This test can probably become something like:
```js
var gs = stream('./fixtures/whatsgoingon/**/*.txt', [], { cwd: __dirname, });

try {
  gs.write({});
} catch (err) {
  expect(err).toExist();
  done();
}
```
However, if we are going to have a test like that we should probably have an inverse that uses `gs.push(globObj)` successfully (because `.push` is the companion to `.write`)